docs/docsite/rst/community/documentation_contributions.rst

```diff
@@ -0,0 +1,166 @@
+#. Be patient while Ansibot, our automated script, adds labels, pings the docs maintainers, and kicks off a CI testing run.
+#. Keep an eye on your PR - the docs team may ask you for changes.
+
+Opening a new issue and/or PR
+=============================
+
+If the problem you've noticed is too complex to fix with the ``Edit on GitHub`` option, please open an issue and/or a PR on the ``ansible/ansible`` repo.
+
+A great documentation GitHub issue or PR includes:
+
+- a specific title
+- a detailed description of the problem (even for a PR - it's hard to evaluate a suggested change unless we know what problem it's meant to solve)
+- links to other information (related issues/PRs, external documentation, pages on docs.ansible.com, etc.)
+
+Before you open a complex documentation PR
+==========================================
+
+If you make multiple changes to the documentation, or add more than a line to it, before you open a pull request, please:
+
+#. Check that your text follows our :ref:`style_guide`.
+#. Test your changes for rST errors.
+#. Build the page, and preferably the entire documentation site, locally.
+
+To work with documentation on your local machine, you need the following packages installed:
+
+.. code-block:: none
+
+   - libyaml
+   - pyyaml
+   - nose
+   - six
+   - tornado
+   - pyparsing
+   - gcc
+   - jinja2
+   - rstcheck
+   - sphinx
+
+.. note::
+
+    On macOS with Xcode, you may need to install ``six`` and ``pyparsing`` with ``--ignore-installed`` to get versions that work wth ``sphinx``.
+
+Testing the documentation locally
+---------------------------------
+
+To test an individual file for rST errors:
+
+.. code-block:: bash
+
+   rstcheck changed_file.rst
+
+Building the documentation locally
+----------------------------------
+
+Building the documentation is the best way to check for errors and review your changes. Once `rstcheck` runs with no errors, navigate to ``ansible/docs/docsite`` and then build the page(s) you want to review.
+
+Building a single rST page
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+To build a single rST file, you have two options:
+
+1. Building an rST file with the make utility:
+
+.. code-block:: bash
+
+   make htmlsingle rst=path/to/your_file.rst
+
+For example:
+
+.. code-block:: bash
+
+   make htmlsingle rst=community/documentation_contributions.rst
+
+This method compiles all the links but provides minimal log output.
+
+.. note::
+
+    ``make htmlsingle`` adds ``rst/`` to the beginning of the path you provide in ``rst=``, so you can't type the filename with autocomplete. If you run ``make htmlsingle`` from the ``docs/docsite/rst/`` directory, you'll get ``make: *** No rule to make target `htmlsingle'.  Stop.`` If you run it from ``docs/docsite/`` and provide the full path, you'll get ``sphinx-build: error: cannot find files ['rst/rst/community/documentation_contributions.rst']``.
+
+2. Building an rST file with sphinx-build:
```

Comment:
I wonder if you could his should be marked as advanced? I've never use it.

Maybe move to the end of the doc under and `advanced` section